frontend/src/Editor/WidgetManager/widgetConfig.js

```diff
@@ -5111,8 +5111,8 @@ ReactDOM.render(<ConnectedComponent />, document.body);`,
   },
   {
     name: 'Kanban',
-    displayName: 'Kanban',
-    description: 'Kanban',
+    displayName: 'Kanban Board',
```

Comment:
Hey @Chetax , please revert this change; the `displayName` value should not be altered. It should be back to 'Kanban'. You can keep the changes in the `description` value.